platformio.ini

```diff
@@ -14,7 +14,9 @@ lib_deps =
 
 [appmod_leds]
 build_flags = 
-  -D APPMOD_LEDS -D USERMOD_ARTNET -D USERMOD_DDP
```

Comment:
Why this change? Surely if we have the option to enable/disable at runtime why exclude these from the build? Especially as they have no library dependencies so have little impact of image size or compile time

Reply:
Just temporary, as a consequence of me merging too soon in main 😉, if I run this I got all kinds of messages so I propose until the mods are working fine we leave it of in main.

Reply:
Maybe set the mod to be present, but disabled at runtime by default?